pages/members/index.js

```diff
@@ -0,0 +1,98 @@
+import Member from "../../components/Member"
+import React, { useState } from "react";
+import Banner from "../../components/Banner"
+import Link from "next/link"
+import Head from "next/head"
+
+export default function MemberPage({ memberData }) { 
+
+  const memberElements = memberData.map((element, index) => {
+    return <Member {...element} key={element.id}/>
+  })
+
+  return (
+    <div className="px-8 bg-gray-900 min-h-screen">
+      <Head>
+        <title>DCV Members</title>
+        <meta name="author" content="toast.eth"/>
+      </Head>
+      <div >
+        <header className="flex justify-center">
+          <Link href="/">
+            <a>
+              <Banner />
+            </a>
+          </Link>
+        </header>
+          <div className="flex flex-wrap justify-center px-[130px] pb-[130px] mt-16 gap-10 font-poppins">
```

Comment:
No need to change but a nit comment since I see a hardcode px value :)

Hardcode pixel value for margin, spacer, and padding is generally a bad idea since it makes it harder to scale and reason when dealing with responsive design.

Reply:
ah, here I used it because it was based on the fixed size of the card expanding, so would potentially have problems if the spacing became responsive through relative values